build/klh10/start

```diff
@@ -21,11 +21,17 @@ stop() {
     echo "OK"
 }
 
+chaosnet() {
+    (sleep 2; tools/cbridge/cbridge -vs -c build/cbridge.conf >cbridge.log 2>&1) &
```

Comment:
I wouldn't recommend -vs, this gives so much (irrelevant) chatter that it might slow the net down, not to mention filling the disk. If what you really want is a "startup printout", that can be arranged - or tell me what you're really after?

Reply:
Ideally I'd like to see a small amount of logging that is useful to see what is happening on the network.  Maybe a line for each connection started or dropped.

Reply:
Connections only exist in the NCP, not in the plain bridge. Or do you mean "connection" as in "the first arrival of traffic on a link"?

Another option is using the hostat program, if you're just interested in seeing how much traffic there is (and how many packets are dropped etc)?

Reply:
I'm intentionally vague here.  I'm not sure exactly what I want, but it's good to see e.g. incoming supdup connections.

Reply:
I would suggest leaving the -vs out until we figure this out, in the interest of wear and tear on SD cards. Sustainability matters, even in the small.

Supdup connections would not be incoming to cbridge, but to ITS. You can probably do an iptables/netfilter rule to find SUPDUP RFCs, if you like.

Reply:
If you only want Supdup connections (which DO come to cbridge when using the NCP), try "trace on" for the ncp config line and see if that's more like what you wanted? Still gives a lot of output I guess, and iptables/netfilter rules are easier to tweak/rate limit etc.

Reply:
It's not overly important, so let's figure it out later.  If the need comes up.